docs/ref/models/fields.txt

```diff
@@ -376,6 +376,32 @@ this field's index, if this field is indexed. The default is the project's
 :attr:`~Options.db_tablespace` of the model, if any. If the backend doesn't
 support tablespaces for indexes, this option is ignored.
 
+``db_default``
+--------------
+
+.. versionadded:: 5.0
+
+.. attribute:: Field.db_default
+
+The default value for the field in the database. This can be a literal value or
+a database function, such as :class:`~django.db.models.functions.Now`::
+
+    created = DateTimeField(db_default=Now())
+
+More complex expressions can be used, as long as they are made from literals
+and database functions::
+
+    month_due = DateField(db_default=TruncMonth(Now() + timedelta(days=90)))
+
+However, a database default may not reference another field or model. For
+example, this is invalid::
```

Comment:
We don't need "however" since it isn't counter to the prior description. Also stick to plural.

```suggestion
Database defaults may not reference other fields or models. For example, this
is invalid::
```